### PR TITLE
Update endoscopic tool segmentation on pre-trained weights loader

### PR DESCRIPTION
**Pull request by binliunls**
Remove the `CheckpointLoader` in the `train.json` and add an explanation about how to load pre-trained encoder weights to the bundle.

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Comment by binliunls** [comment]
Hi @SachidanandAlle and @tangy5 ,
     Could you please take a look at this PR to see if it fixes the issue in MONAI label?

Thanks in advance.

**Review by SachidanandAlle — APPROVED** [review]

**Review by yiheng-wang-nv — APPROVED** [review]
